site/content/docs/howto/supervisor/configure-supervisor-with-github.md

```diff
@@ -17,9 +17,6 @@ The Supervisor is an [OpenID Connect (OIDC)](https://openid.net/connect/) issuer
 This guide shows you how to configure the Supervisor so that users can authenticate to their Kubernetes
 cluster using their credentials from [GitHub.com](https://github.com) or [GitHub enterprise server](https://docs.github.com/en/enterprise-server@latest/admin/overview/about-github-enterprise-server).
 
-Currently, Pinniped supports GitHub API version `2022-11-28` ([ref](https://docs.github.com/en/rest/about-the-rest-api/api-versions?apiVersion=2022-11-28)).
```

Comment:
Feels like this should be public information in case the GitHub API does change.

Reply:
nevermind, this was just moved below.